static/templates/search_operators.hbs

```diff
@@ -0,0 +1,239 @@
+    <div class="modal-body" data-simplebar data-simplebar-auto-hide="false">
+        <div id="operators-instructions">
+            <table class="table table-striped table-condensed table-rounded table-bordered help-table">
+                <thead>
+                    <tr>
+                        <th>{{t "Operator" }}</th>
+                        <th>{{t "Effect" }}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="operator">
+                            {{#tr}}
+                                stream:<z-value>stream</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                        <td class="definition">
+                            {{#tr}}
+                                Narrow to messages on stream <z-value>stream</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="operator">
+                            {{#tr}}
+                                topic:<z-value>topic</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                        <td class="definition">
+                            {{#tr}}
+                                Narrow to messages with topic <z-value>topic</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="operator">
+                            {{#tr}}
+                                pm-with:<z-value>email</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                        <td class="definition">
+                            {{#tr}}
+                                Narrow to private messages with <z-value>email</z-value>
+                                {{#*inline "z-value"}}<span class="operator_value">{{> @partial-block}}</span>{{/inline}}
+                            {{/tr}}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="operator">
+                            {{#tr}}
+                                group-pm-with:<z-value>email</z-value>
```

Comment:
Maybe we should add these within the <a href = 'https://github.com/zulip/zulip/blob/87fc2bbe50da15ebcbecfb710e1ef928224bcaa0/tools/lib/capitalization.py#L11'><code>IGNORED_PHRASES</code></a> as well ?